README.md

```diff
@@ -2,17 +2,46 @@
 
 This is code to support the NEXT-D study.
 
-by Al'ona Furmanchuk
-with Dan Connolly
-
- - for MS SQL Server: `NextDcodeForTable1_studyAndDMsampleExtraction.sql`
- - for Oracle: `NextDvariableExtractionOracleTable1GPC.sql`
+Inital code by Al'ona Furmanchuk with Dan Connolly.
+All contributors:
+ - Al'ona Furmanchuk <alona.furmanchuk@northwestern.edu>, <furmanchuk@icnanotox.org>
+ - Brennan Connolly <bconnolly@kumc.edu>
+ - Dan Connolly <dckc@madmode.com>, <dconnolly@kumc.edu>
+ - George Kowalski <gkowalski@mcw.edu>
+ - Mei Liu <meiliu@kumc.edu>
+ - Alex Stoddard <astoddard@mcw.edu>
 
 See also:
 
  - [NEXT-D Data Request Detail](https://informatics.gpcnetwork.org/trac/Project/attachment/ticket/539/NEXT-D_Request%20for%20Data_Detailed_12.1.16.docx) draft of Nov 15
  - [Ticket #546](https://informatics.gpcnetwork.org/trac/Project/ticket/546)
 
+
+
+## Implemenation overview
+
+NEXT-D query code targets PCORNET CDM implementations, originally developed against a SQLServer (see SQLServer_impl) CDM by Al'ona Furmanchuk and then ported to Oracle (see Oracle_impl).
```

Comment:
perhaps make "PCORNET CDM" a link? Everybody in GPC knows what it is, but other readers might not.